.github/workflows/test.yml

```diff
@@ -137,6 +140,7 @@ jobs:
           
           download_tinygranite() {
             python -c "from transformers import pipeline, AutoTokenizer; pipeline('text-generation', model='ibm-ai-platform/micro-g3.3-8b-instruct-1b'); tokenizer=AutoTokenizer.from_pretrained('ibm-ai-platform/micro-g3.3-8b-instruct-1b')"
+            python -c "from transformers import pipeline, AutoTokenizer; pipeline('text-generation', model='ibm-ai-platform/micro-g3.3-8b-instruct-1b'); tokenizer=AutoTokenizer.from_pretrained('ibm-ai-platform/micro-g3.3-8b-instruct-1b-FP8')"
```

Comment:
Why is the pipeline initialized with `'text-generation', model='ibm-ai-platform/micro-g3.3-8b-instruct-1b` and the tokenizer with `ibm-ai-platform/micro-g3.3-8b-instruct-1b-FP8`?